tests/test_visitors/test_ast/test_classes/test_getter_setter.py

```diff
@@ -0,0 +1,158 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from wemake_python_styleguide.violations.oop import (
+    UnpythonicGetterSetterViolation,
+)
+from wemake_python_styleguide.visitors.ast.classes import WrongClassVisitor
+
+module_getter_and_setter = """
+attribute = 1
+
+def get_attribute():
+    ...
+
+def set_attribute():
+    ...
+"""
+
+static_getter_and_setter = """
+attribute = 1
+
+class Test(object):
+    @staticmethod
+    def get_attribute():
+        ...
+
+    @staticmethod
+    def set_attribute():
+        ...
+"""
+
+property_getter_and_setter = """
+class Test(object):
+    def __init__(self):
+        self.attribute = 1
+
+    @property
```

Comment:
I am not sure about this example though. Why? Because the mist pythonic way to do it is:

```python
class Test(object):
    def __init__(self):
        self._attribute = 1

    @property
    def attribute(self):
        ...

    @attribute.setter
    def attribute(self, attribute_value):
        ...
```

Let's forbid this naming even for properties, please.